packages/pangraph/src/pangraph/pangraph_node.rs

```diff
@@ -41,12 +41,12 @@ impl PangraphNode {
     }
   }
 
-  pub fn len(&self) -> usize {
```

Comment:
I this this function had a bug. But due to possibly having circular genomes, the length of a node cannot always be determined by only start-end positions